doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -452,7 +452,9 @@ Modem libraries
 
 * :ref:`pdn_readme` library:
 
-  * Added the :c:func:`pdn_dynamic_params_get_v6` function to get PDN parameters for IPv6-only.
+  * Deprecated the :c:func:`pdn_dynamic_params_get` function. Use the :c:func:`pdn_dynamic_info_get` function instead.
+
+  * Added the :c:func:`pdn_dynamic_info_get` function based on the :c:func:`link_api_pdp_context_dynamic_params_get` function.
```

Comment:
@peknis Is this "added" entry for `pdn_dynamic_info_get()` needed or is the deprecated entry above sufficient? It already mentions `pdn_dynamic_info_get()`. At least I think that mentioning `link_api_pdp_context_dynamic_params_get()` is unnecessary, because it was an modem_shell internal function.

Reply:
Good point @tokangas, I believe the Deprecated entry would be enough. Feel free to remove the Added entry.

Reply:
done